cmflib/cmf.py

```diff
@@ -1725,5 +1730,30 @@ def non_related_args(type:str,args:dict):
         if repo ==type:
             non_related_args=list(set(available_args)-set(dict_repository_args[repo]))
     return non_related_args
- 
-    
+
+
+def get_python_env()-> str:
+    installed_packages = ""
+    python_version = sys.version

```

Comment:
What happens when both pip and conda is installed ?

Reply:
conda gets preference.